app/views/results/common/_annotations.js.erb

```diff
@@ -380,8 +380,9 @@
     positions = null;
     annotation_manager = null;
 
+    // TODO - Need to see if submission_id is required here, or if it breaks something unexpectedly?
     jQuery.ajax({
-      url:  '<%= codeviewer_assignment_submission_result_path(uid: @uid) %>',
+      url:  '<%= codeviewer_assignment_submission_result_path(uid: @uid, submission_id: @submission.id) %>',
```

Comment:
I marked this because I had to add this in for it to work. I've checked it in the other view for the admin to make sure the view doesn't some drastically break, and I had no problem loading it as an admin under the submissions viewer.

I noted this because rails passes `submission_id` in from the controller normally, but in our case this is absent now, the question then is it potentially dangerous to use this? It assumes that the edit part of the controller will never change the `@submission` to be something else.